### ISSUE TITLE
Installation as a dependency

### ISSUE THREAD

**Issue by bgailleton** [TopoToolbox/pytopotoolbox#67]
Hi, Now `graphflood`'s PR is ready to merge, I'd like to add pytopotoolbox as a dependency to my own [tools](https://github.com/bgailleton/scabbard). What is the best practice and/or future plan for automating installation?
The docs refer to `.whl` but I could not find them. On the other hand if you have a CI that builds them it is straightforward to upload them to `pypi` for `pip` installs (literally a couple of command lines). If you want to build a sophisticated CI, there is also `conda-forge` but it requires a bit more work.

**Comment by wkearn** [comment]
I'd like to leave this open to track progress on the installation
process, if that is all right.

**Comment by wkearn** [comment]
Would it be useful if we built wheels and released them here even if they're not available on PyPI? We could pretty easily set up a GitHub Actions workflow to do this either on every commit to `main` or on a regular schedule -- we do weekly releases of libtopotoolbox.

**Comment by bgailleton** [comment]
I mean, from my side I need to be able to point to precompiled wheels so users don't have to compile the code themselves (by experience, do not underestimate the number of user that would put off). So if you set up some sort of release system that publish a wheel, I'll be happy to use it.

Just by curiosity, what is the reasoning behind not directly uploading them to pypi at the same time?

**Comment by wkearn** [comment]
I thought I had replied here a while ago, but maybe I tried to reply via email and that didn't go through.

In any case, I have since come around on uploading pytopotoolbox to PyPI because it will make installation a lot easier.

My main worry had been around making sure that the relationship between versions of libtopotoolbox and pytopotoolbox was clear. However, since we download a tagged version of libtopotoolbox, compile and link it statically to the shared library that scikit-build-core creates for us, I don't think we actually need to worry about the different versions. We will just use the tag of libtopotoolbox that is present in CMakeLists.txt. We may want to do this differently in the future.

I do want to release 0.x.x versions for now, just because I don't think we able to commit to a stable API yet.
